### PR TITLE
show page fix + carousel started

### PR DESCRIPTION
**Pull request by TheRealCL**
PLEASE VERIFY

### CODE REVIEW

**Review by MBOZAR — APPROVED** [review]
Approve, the code on show page was duplicated

File: app/views/nvts/show.html.erb [hunk 1]
```diff
@@ -26,25 +26,3 @@
     <p><strong>Interested in collecting this item?</strong></p>
   <%= link_to 'Collect', transaction_path %>
```
Comment (Rox1610) [inline app/views/nvts/show.html.erb:27]: <%= link_to 'Collect', new_nvt_transaction_path(@nvt) %>

**Review by Joe-woods — APPROVED** [review]

**Review by Rox1610 — CHANGES_REQUESTED** [review]
modify path on show page

**Review by MBOZAR — CHANGES_REQUESTED** [review]
please see rox fix thanks

**Review by Rox1610 — APPROVED** [review]
ok

**Review by MBOZAR — APPROVED** [review]
approved

**Review by Joe-woods — APPROVED** [review]

**Pull request merged**